.travis.yml

```diff
@@ -0,0 +1,51 @@
+# The language in this case has no bearing - we are going to be making use of conda for a
+# python distribution for the scientific python stack.
+language: generic
+
+sudo: false
+
+env:
+    global:
+       - CONDA_INSTALL_LOCN="${HOME}/conda"
+
+       # Adds a GH_TOKEN (conda-forge-admin's conda-forge.github.io token)
+       - secure: __FIX_ME__
+
```

Comment:
May need to override Travis CI's logic for git submodules based on this [SO answer]( http://stackoverflow.com/a/24600210 ).